lua/astrocommunity/pack/godot/README.md

```diff
@@ -7,4 +7,10 @@ This plugin pack does the following:
 - Adds `gdscript` treesitter parsers
 - Adds `godot_resource` treesitter parsers
 - Adds `glsl` treesitter parsers
-- Adds [quickgd.nvim](https://github.com/QuickGD/quickgd.nvim)
+- Adds [godot-server.nvim](https://github.com/Cretezy/godot-server.nvim) to open files from Godot in Neovim. Requires configuration in the Godot editor (see [project page](https://github.com/Cretezy/godot-server.nvim))
+- Adds [quickgd.nvim](https://github.com/QuickGD/quickgd.nvim) to provide `:GodotStart` (starts project), `:GodotRun` (shows picker to run scenes), and `:GodotRunLast` (runs the last ran scene)
```

Comment:
```suggestion
```

Reply:
This is too wordy. Our readme shouldn't replace the docs, since the docs might be updated down the road.